docs/library.rst

```diff
@@ -522,9 +522,10 @@ exception.
 Refactorings
 ============
 
-Have a look at ``rope.refactor`` package and its sub-modules.  For
-example for performing a move refactoring you can create a ``Move``
-object like this:
+Have a look at the `rope.refactor` package and its sub-modules.  For
+example for performing a move refactoring you can create an object
+representing this operation (which will be an instance of e.g.
+`MoveMethod`, `MoveModule`, ...) using `create_move`.
```

Comment:
I've made a few adjustments like this one in `library.rst` just to show the benefit of the API docs for readers of the existing docs.

Ideally, this should be done throughout the existing docs, but I'll only do that if I get the green light for the basic idea behind this PR.